memo/js/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Can you remove this?

Reply:
I can, but the value of it is that it makes an IDE use the same spacing settings as Prettier is configured to here. Thoughts?

Reply:
It's just spacing, so I'd prefer not.  Many IDEs infer from prettier files what to do as well. Finally, CI checks for it anyway, so let's axe it

Reply:
Got it. Removed!